keycloak/keycloak-ui
# Fixed helpText for LDAP mapper types -#1482

## Motivation
Closes #1482

## Verification Steps
1. Go to User Federation --> LDAP --> LDAP Mappers
2. Create a new mapper
3. Choose any mapper type
4. Verify that the mapper type helpText changes based on the selected mapper type.

## Checklist:
